src/factory/providers/ethereum/clearpool/data.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We don't use data.json anymore as we have DAPP_TOKEN_DATA DB table

Reply:
We talked about this, and we decided to put it, developers outside of the company should also put it so that we can update the DB for the new provider.